runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -1267,5 +1290,34 @@ private void updateConfig()
 		this.prayerColor = config.prayerColor();
 		this.highlightHerblore = config.highlightHerblore();
 		this.highlightPrayer = config.highlightPrayer();
+		this.inventoryStacks = config.inventoryStacks();
+	}
+
+	public void onItemContainerChanged(ItemContainerChanged itemContainerChangedEvent)
+	{
+		if (config.inventoryStacks() && itemContainerChangedEvent.getContainerId() == InventoryID.INVENTORY.getId())
```

Comment:
`if (this.inventoryStacks && itemContainerChangedEvent.getContainerId() == InventoryID.INVENTORY.getId())`